.run/DiscoveryServiceApplication.run.xml

```diff
@@ -0,0 +1,27 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="DiscoveryServiceApplication" type="SpringBootApplicationConfigurationType" factoryName="Spring Boot">
+    <module name="api-layer.discovery-service.main" />
+    <option name="SPRING_BOOT_MAIN_CLASS" value="org.zowe.apiml.discovery.DiscoveryServiceApplication" />
+    <option name="ACTIVE_PROFILES" value="https" />
+    <option name="ALTERNATIVE_JRE_PATH" />
+    <additionalParameters>
+      <param>
+        <option name="enabled" value="true" />
+        <option name="name" value="spring.config.additional-location" />
+        <option name="value" value="file:./config/local/discovery-service.yml" />
+      </param>
+      <param>
+        <option name="enabled" value="true" />
+        <option name="name" value="apiml.discovery.staticApiDefinitionsDirectories" />
+        <option name="value" value="config/local/api-defs;../api-defs" />
```

Comment:
is ../api-defs part of the project?

Reply:
no it isn't. Discovery service doesn't care if the directory doesn't exist.

Reply:
it gives a warning